charts/astronomer/templates/houston/houston-configmap.yaml

```
@@ -206,6 +206,11 @@ data:
         postgresql:
           enabled: false
 
+        # If security context constraints are enabled, enable SCCs for airflow-chart
```

I wonder if these should be golang template comments, not yaml comments, so we don't end up with just a floating text in the yaml output. Not a huge deal though, and seems less readable.
```suggestion
        {{- /* If security context constraints are enabled, enable SCCs for airflow-chart */ }}
```

---

I think we should keep it consistent with the other comments, but I move it into the helm conditional